src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -1092,7 +1100,15 @@ public void addCollaborators(GHUser... users) throws IOException {
      *             the io exception
      */
     public void addCollaborators(Collection<GHUser> users) throws IOException {
-        modifyCollaborators(users, "PUT", null);
+        modifyCollaborators(users, "PUT", (GHOrganization.Permission) null);
+    }
+
+    /**
+     * @deprecated #addCollaborators(Collection, GHOrganization.RolePermission)
+     */
+    @Deprecated
+    public void addCollaborators(Collection<GHUser> users, GHOrganization.Permission permission) throws IOException {
+        modifyCollaborators(users, "PUT", permission);
```

Comment:
Side note: I think the `/collaborators` endpoint as a whole is deprecated by GitHub.  I think they still work, but I can't find them in the docs... So maybe you don't need to convert these methods, but rather find what the new ones are. 
```suggestion
        addCollaborators(users, GHOrganization.RolePermission.from(permission.name().toLowerCase());
```

Reply:
seems like it's still there: https://docs.github.com/en/rest/reference/collaborators

Reply:
Grr.  😭 
There are times I hate this API.  Please add that link to the JavaDocs, maybe for `modifyCollaborators(...)` so that people can find it easier next time.